drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -2184,10 +2184,15 @@ _FORCE_INLINE_ static void _fill_std140_variant_ubo_value(ShaderLanguage::DataTy
 			Transform2D v = value;
 			GLfloat *gui = (GLfloat *)data;
 
+			//in std140 members of mat2 are treated as vec4s
 			gui[0] = v.elements[0][0];
 			gui[1] = v.elements[0][1];
-			gui[2] = v.elements[1][0];
-			gui[3] = v.elements[1][1];
+			gui[2] = 0;
```

Comment:
The comment is good, thanks. If we don't need those indices, don't we risk a small performance loss by setting them to 0 though? You can just mention that since the members are vec4s, we skip the y and w coordinates.

Reply:
@reduz seems to say it doesn't make much difference, so I'll merge as is :)